.mise/tasks/repo/setup

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env sh
 
 # mise description="setup dependencies for the repo"
+# mise tools=["python@latest", "poetry"]
```

Comment:
Not sure if we want `latest` or something else for Python but not holding up the PR over it.

Reply:
yea, good call, lemme pin it to 3.10 actually since that's the most stable with the pytorch libs

Reply:
fixed